src/Pim/index.tsx

```diff
@@ -395,6 +395,25 @@ class Pim extends React.PureComponent {
     const { collectionsAddressBook, collectionsCalendar, collectionsTaskList, addressBookItems, calendarItems, taskListItems } = itemsSelector(this.props);
     const [contacts, events, tasks] = itemValuesSelector(this.props);
 
+    const birthdays = contacts.filter((c) => c.bday).map((c) => {
+      let { year, month, day } = c.bday._time;
+      year = year || '2020';
```

Comment:
Another thing, does this actually work with birthdays without the year set? It's great if it does. What's the type of `bday`? (Actualy type, not typescript type). I remember that it could change and thus `_time` wouldn't always be available. If it doesn't, then you can just change `bday` below to return the correct type. (`getFirstPropertyValue<ICAL.Time>(...)`) and then work with it. Though I remember that we had to hack around it in the iOS client. Take a look at what we did there: https://github.com/etesync/ios/blob/master/src/sync/helpers.ts#L417

Reply:
Having given it more thought, I'm pretty sure it's indeed the case, and that you should use `parsDate` from the link. I remember opening a ticket about it on ICAL.js's bug tracker.